sdk/containerregistry/azure-containerregistry/azure/containerregistry/_helpers.py

```diff
@@ -18,8 +20,13 @@
 from azure.core.exceptions import ServiceRequestError
 from ._generated.models import OCIManifest
 
+from azure.mgmt.containerregistry import ContainerRegistryManagementClient
```

Comment:
This SDK doesn't have a dependency on the mgmt SDK does it?

Reply:
It does, the mgmt client is used to import images.